hex/spec/dependabot/hex/file_updater/lockfile_updater_spec.rb

```diff
@@ -220,6 +220,26 @@
       end
     end
 
+    context "with a mix.exs that opens another file in a required file" do
+      let(:mixfile_fixture_name) { "loads_file_with_require" }
+      let(:lockfile_fixture_name) { "exact_version" }
+      let(:files) { [mixfile, lockfile, support_file] }
+      let(:support_file) do
+        Dependabot::DependencyFile.new(
+          name: "module_version.ex",
+          content: fixture("support_files", "module_version"),
+          support_file: true
+        )
+      end
+
+      it "updates the dependency version in the lockfile" do
+        expect(updated_lockfile_content).to include %({:hex, :plug, "1.4.3")
+        expect(updated_lockfile_content).to include(
+          "236d77ce7bf3e3a2668dc0d32a9b6f1f9b1f05361019946aae49874904be4aed"
```

Comment:
Where does this string come from? Maybe we could assign it to a variable that explains what it represents?

```
let(:expected_something_something) { "236d77ce7bf3e3a2668dc0d32a9b6f1f9b1f05361019946aae49874904be4aed" }
```

Reply:
this was copied from the tests above it. I'll leave it to you all to decide if you want to extract it out into a variable though.

Reply:
Ah, right, I think we should but we can follow up on that separately, thanks!